test/test_cluster.py

```diff
@@ -0,0 +1,24 @@
+# pylint: skip-file
+from __future__ import absolute_import
+
+import pytest
+
+from kafka.cluster import ClusterMetadata
+from kafka.protocol.metadata import MetadataResponse
+
+import kafka.common as Errors
```

Comment:
This is confusing... why import common as Errors? Why not directly import errors as errors or common as common?

Reply:
good catch! I removed that line as it is not needed. But generally re: kafka.common -- that line was copy-pasta from a file that was written a long time ago before kafka.common was split into kafka.errors and kafka.structs . The only reason to keep kafka.common is to prevent breaking user code that was written before the change. But for our library code we should use kafka.errors directly.

Reply:
That's what I thought... I finished the library cleanup here: https://github.com/dpkp/kafka-python/pull/1509 so hopefully this won't happen again...